resources/kubernetes/operator/helm/values.yaml

```diff
@@ -1,24 +1,28 @@
+# For advanced configuration options, refer to the [official OpenTelemetry Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml)
+
 opentelemetry-operator:
   manager:
     extraArgs:
       - --enable-go-instrumentation
   admissionWebhooks:
     certManager:
-      enabled: false
+      enabled: false # For production environments, it is [recommended to use cert-manager for better security and scalability](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator#tls-certificate-requirement).
+
 
 autoGenerateCert:
-  enabled: true
-  recreate: true
+  enabled: true # Enable/disable automatic certificate generation. Set to false if manually managing certificates.
+  recreate: true # Force certificate regeneration on updates. Only applicable if autoGenerateCert.enabled is true.
 
 crds:
-  create: true
+  create: true # Install the OpenTelemetry Operator CRDs.
 
 defaultCRConfig:
   image:
     repository: "docker.elastic.co/beats/elastic-agent"
     tag: "8.16.0-SNAPSHOT"
   targetAllocator:
-    enabled: false
+    enabled: false # Enable/disable the Operator's Target allocator.
+                   # Refer to: https://github.com/open-telemetry/opentelemetry-operator/tree/main/cmd/otel-allocator
```

Comment:
nit: dont like it so much that is in 2 lines
Maybe to move Refer line before targetAllocator and line 23?

Reply:
Sounds good to me, but I think we will need to do this in another PR to keep the history aligned with main